ttml.go

```diff
@@ -288,6 +289,10 @@ func (d TTMLInDuration) duration() time.Duration {
 	if d.framerate > 0 {
 		return d.d + time.Duration(float64(d.frames)/float64(d.framerate)*1e9)*time.Nanosecond
 	}
+	if d.tickrate > 0 {
```

Comment:
Why not compute the value once and for all [here](https://github.com/asticode/go-astisub/blob/master/ttml.go#L266) ? Is the `tickrate` value not populated at that time ?

Reply:
Unfortunately, it is not populated at that time.